doc/docs/dev-guide/faq.md

```diff
@@ -0,0 +1,184 @@
+# FAQ
+
+Frequently asked questions and useful information.
+
+---
+
+## Useful utilities
+
+### Swagger
+
+* [Swagger](https://swagger.io/)
+* [Swagger Editor](https://editor.swagger.io/)
+
+### PlantUML
+
+* [PlantUML Language Reference Guide](http://plantuml.com/guide)
+* [PlantText - Online Editor](https://www.planttext.com/)
+
+---
+
+## How to sign a release using GPG?
+
+A good example of a release signature can be found on the [link](https://wiki.debian.org/Creating%20signed%20GitHub%20releases). However, 5 and 6 items are described separately.
+
+### Creating a key pair
+
+```bash
+gpg --full-generate-key
+```
+
+During the execution of this command, you will need to enter additional data:
+
+* Please select what kind of key you want: **1**
+* What keysize do you want? (3072): **4096**
+* Please specify how long the key should be valid: **0**
+* Key is valid for? (0): **0**
+* Is this correct? (y/N): **y**
+* Real name: **LPVS**
+* Email address: **o.kopysov@samsung.com**
```

Comment:
How about to use more common examples to depersonalize guide. For example, like in part "DCO via the command line": "John Doe" johndoe@example.com